wodles/aws/aws_s3.py

```diff
@@ -537,6 +537,7 @@ def __init__(self, reparse, access_key, secret_key, profile, iam_role_arn,
         self.prefix_regex= re.compile("^\d{12}$")
         self.check_prefix = False
         self.date_format = "%Y/%m/%d"
+        self._db_date_format = "%Y%m%d"
```

Comment:
```suggestion
        self.db_date_format = "%Y%m%d"
```
Both `date_format` and `db_date_format` should have the same scope and because of that they should be both public or private.